report/index.adoc

```diff
@@ -67,7 +68,8 @@ We configured the New Relic agent in our Dockerfile, ensuring consistent logging
 === Security Assessment
 
 // Brief results of the security assessment and description of how the system security was hardened.
-*OLIVER*
+The team has uncovered three major security threats via risk assessment, the risks are described in the <<Risk Assessments>> chapter of the Appendix. The team has addressed these threats, but due to the time constraint of the project as well as low probability of an attack, the tasks to implement fixes was under prioritized. 
```

Comment:
Måske et punktum efter "assessment", eller word starten af næste sætning lidt anderledes. F.eks. "assessment, and these are described...". Something like that.

Jeg kan også læse mig til at "underprioritized" er i ét ord.